src/meta/geo/layer.ts

```diff
@@ -0,0 +1,63 @@
+import { BurnedAreaKey } from './burnedAreaSource'
+import { ForestKey, HansenPercentage } from './forest'
+import { ProtectedAreaKey } from './protectedAreaSource'
+
+export type LayerConfig = {
+  mapId: string
+  palette: Array<string>
+  year?: number
+  scale?: number
+  citation?: string
+}
+
+export interface LayerSource {
+  key: ForestKey | ProtectedAreaKey | BurnedAreaKey
+  options?: {
+    gteTreeCoverPercent?: number
+    assetId?: string
+    year?: number
+    agreement?: {
+      layers: Array<LayerSource>
+      gteAgreementLevel: number
+    }
+  }
+}
+
+export interface Recipe {
+  layers: Array<ForestKey>
+  gteHansenTreeCoverPerc?: HansenPercentage
+  forestAreaDataProperty: string
+  recipeLabel: string
+}
+
+export type LayerMetadata = {
+  scale?: number
+  palette?: Array<string>
+  citation?: string
+  forestAreaDataProperty?: string
+}
+
+export type LayerOptions = {
+  minTreeCoverPercentage?: Array<number>
+  agreementLayerOptions?: { agreementLevels: Array<number>; reducerScales: Array<number> }
```

Comment:
I would call this agreementLayer without options postfix. it's implicit by the type definitions

Reply:
Yes, good point. I updated it.